README.md

```diff
@@ -8,6 +8,22 @@
 
 <img width="200" align="right" alt="OpenEBS Logo" src="https://raw.githubusercontent.com/cncf/artwork/HEAD/projects/openebs/stacked/color/openebs-stacked-color.png" xmlns="http://www.w3.org/1999/html">
 
+## ⚠️ Deprecation Notice: Helm Chart Registry Migration
+
+The Helm chart registry at "https://openebs.github.io/charts" is deprecated and will be moved to a new location soon.
+
+**Old Registry URL:** https://openebs.github.io/charts  
+**New Registry URL:** https://openebs.github.io/openebs  
+
+To ensure seamless access to OpenEBS Helm charts, please update your configurations to use the new registry URL.
+
```

Comment:
Avoid using the term "please" throughout the documentation.

Reply:
Done.